src/macros.jl

```diff
@@ -1429,13 +1434,18 @@ end
 
 # TODO: Add a docstring.
 macro NLobjective(m, sense, x)
-    if sense == :Min || sense == :Max
-        sense = Expr(:quote,sense)
+    if sense == :Min
+        sense = MOI.MinSense
+    elseif sense == :Max
+        sense = MOI.MaxSense
+    else
+        # Refers to a variable that holds the sense.
```

Comment:
It might improve readability to create a funtion like `sense_from_symbol` doing the if-then-else and used in both macros

Reply:
I added TODOs. IMO just moving this if/else into a method would be confusing because of the logic in
```julia
# Refers to a variable that holds the sense.
sense = esc(sense)
```
Needs a bigger refactor to properly address.